lib/licenses.nix

```diff
@@ -1223,6 +1223,13 @@ in mkLicense lset) ({
     fullName = "The Unlicense";
   };
 
+  unrar = {
+    fullName = "UnRAR - free utility for RAR archives: License for use and distribution of FREEWARE version";
+    url = "https://www.rarlab.com/rar/unrarsrc-7.0.9.tar.gz";
```

Comment:
Maybe we could link to <https://fedoraproject.org/wiki/Licensing:Unrar>, which has the text. I think this would be okay since we use SPDX and OSI links for some licences. On the other had it does have editorial commentary :)